addon/components/admin-input.js

```diff
@@ -1,18 +1,21 @@
 import Ember from 'ember';
 
-var set = Ember.set;
-var get = Ember.get;
-var computed = Ember.computed;
+const set = Ember.set;
+const get = Ember.get;
+const computed = Ember.computed;
```

Comment:
Is `ember build` giving you that error? I just pulled down your branch and added it without any compilation issue.